db/src/main/kotlin/edu/artic/db/models/ArticTour.kt

```diff
@@ -131,6 +132,29 @@ data class ArticTour(
         return this.title
     }
 
+    /**
+     * Returns the translations including english language
+     */
+    val translationWithEnglish: List<Translation>
+        get() {
+            return mutableListOf(this.toEnglishTranslation()).apply { addAll(translations) }
+        }
+
+}
+
+/**
+ * Builds english translation for [ArticTour]
```

Comment:
Edit comment to make it clear that we are returning, essentially, a copy of `this`.

Reply:
Edited.